ete3/coretype/tree.py

```diff
@@ -1552,7 +1552,7 @@ def sort_descendants(self, attr="name"):
             if not n.is_leaf():
                 n.children.sort(key=lambda x: str(sorted(node2content[x])))
 
-    def get_cached_content(self, store_attr=None, container_type=set, _store=None):
+    def get_cached_content(self, store_attr=None, container_type=set, _store=None, descendant_leaves=True):
```

Comment:
can we call the flag "leaves_only=True"? I think it is more consistent with other functions.